ergo-lib/src/wallet/signing.rs

```diff
@@ -170,6 +171,7 @@ pub fn sign_transaction(
     prover: &dyn Prover,
     tx_context: TransactionContext,
     state_context: &ErgoStateContext,
+    tx_hints: &TransactionHintsBag,
```

Comment:
Let's make this `tx_hints: Option<&TransactionHintsBah>` as well. As it is done in `wallet`.